.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: node_js
+sudo: false
 node_js:
-  - 0.10
+  - 0.12
 
 branches:
   only:
     - master
 
 before_script:
-  - npm install -gq grunt-cli bower coveralls
-  - bower install
+  - npm install -gq coveralls
```

Comment:
We usually don't need to install cli tool globally as npm include the local package binaries in the PATH of npm scripts.